packages/governance-modules/cannonfile.toml

```diff
@@ -0,0 +1,92 @@
+name = "synthetix-governance"
+description = "Governance cotracts for on chain voting"
+version = "0.0.1"
```

Comment:
same for binding to the package.json value

if we want to keep cannon versino separate from package for the time being, then please disregard this comment.

Reply:
I think we should use another version for now, for the case that we build the same version of `governance-modules` with a different cannon version (which we are probably going to publish a lot on the following weeks)